x-pack/plugins/security_solution/public/detection_engine/rule_management_ui/components/maintenance_window_callout/maintenance_window_callout.test.tsx

```diff
@@ -0,0 +1,223 @@
+    { gte: '2023-10-27T10:36:14.028Z', lte: '2023-10-27T10:37:00.000Z' },
+    { gte: '2023-11-03T11:36:14.028Z', lte: '2023-11-03T11:37:00.000Z' },
+    { gte: '2023-11-10T11:36:14.028Z', lte: '2023-11-10T11:37:00.000Z' },
+    { gte: '2023-11-17T11:36:14.028Z', lte: '2023-11-17T11:37:00.000Z' },
+    { gte: '2023-11-24T11:36:14.028Z', lte: '2023-11-24T11:37:00.000Z' },
+    { gte: '2023-12-01T11:36:14.028Z', lte: '2023-12-01T11:37:00.000Z' },
+    { gte: '2023-12-08T11:36:14.028Z', lte: '2023-12-08T11:37:00.000Z' },
+    { gte: '2023-12-15T11:36:14.028Z', lte: '2023-12-15T11:37:00.000Z' },
+    { gte: '2023-12-22T11:36:14.028Z', lte: '2023-12-22T11:37:00.000Z' },
+    { gte: '2023-12-29T11:36:14.028Z', lte: '2023-12-29T11:37:00.000Z' },
+    { gte: '2024-01-05T11:36:14.028Z', lte: '2024-01-05T11:37:00.000Z' },
+    { gte: '2024-01-12T11:36:14.028Z', lte: '2024-01-12T11:37:00.000Z' },
+    { gte: '2024-01-19T11:36:14.028Z', lte: '2024-01-19T11:37:00.000Z' },
+    { gte: '2024-01-26T11:36:14.028Z', lte: '2024-01-26T11:37:00.000Z' },
+    { gte: '2024-02-02T11:36:14.028Z', lte: '2024-02-02T11:37:00.000Z' },
+    { gte: '2024-02-09T11:36:14.028Z', lte: '2024-02-09T11:37:00.000Z' },
+    { gte: '2024-02-16T11:36:14.028Z', lte: '2024-02-16T11:37:00.000Z' },
+    { gte: '2024-02-23T11:36:14.028Z', lte: '2024-02-23T11:37:00.000Z' },
+    { gte: '2024-03-01T11:36:14.028Z', lte: '2024-03-01T11:37:00.000Z' },
+    { gte: '2024-03-08T11:36:14.028Z', lte: '2024-03-08T11:37:00.000Z' },
+    { gte: '2024-03-15T11:36:14.028Z', lte: '2024-03-15T11:37:00.000Z' },
+    { gte: '2024-03-22T11:36:14.028Z', lte: '2024-03-22T11:37:00.000Z' },
+    { gte: '2024-03-29T11:36:14.028Z', lte: '2024-03-29T11:37:00.000Z' },
+    { gte: '2024-04-05T10:36:14.028Z', lte: '2024-04-05T10:37:00.000Z' },
+    { gte: '2024-04-12T10:36:14.028Z', lte: '2024-04-12T10:37:00.000Z' },
+    { gte: '2024-04-19T10:36:14.028Z', lte: '2024-04-19T10:37:00.000Z' },
+  ],
+  id: '5eafe070-e030-11ed-ac81-bfcb4982b4fd',
+  status: 'upcoming',
+  expiration_date: '2024-04-21T10:36:26.999Z',
+  r_rule: {
+    dtstart: '2023-04-21T10:36:14.028Z',
+    tzid: 'Europe/Amsterdam',
+    freq: 3,
+    interval: 1,
+    byweekday: ['FR'],
+  },
+  created_by: 'elastic',
+  updated_by: 'elastic',
+  created_at: '2023-04-21T10:36:26.999Z',
+  updated_at: '2023-04-21T10:36:26.999Z',
+  event_start_time: '2023-04-28T10:36:14.028Z',
+  event_end_time: '2023-04-28T10:37:00.000Z',
+};
+
+describe('MaintenanceWindowCallout', () => {
+  let appToastsMock: jest.Mocked<ReturnType<typeof useAppToastsMock.create>>;
+
+  beforeEach(() => {
+    jest.resetAllMocks();
+
+    appToastsMock = useAppToastsMock.create();
+    (useAppToasts as jest.Mock).mockReturnValue(appToastsMock);
+  });
+
+  afterEach(() => {
+    cleanup();
+    jest.restoreAllMocks();
+  });
+
+  it('MW callout should be visible if currently there is at least one "running" maintenance window', async () => {
```

Comment:
nit: you already have `MaintenanceWindowCallout` in describe so it can be omitted and the text become `should be visible if currently there is at least one "running" maintenance window`. And the same for the tests below.

Reply:
Good idea. Thanks. Done.